### PR TITLE
Schedule Test Repository tests if not yet done

### PR DESCRIPTION
**Pull request by coolo**
(for this check the checksum of the repo checksums)

### CODE REVIEW

File: openqa-maintenance.py [hunk 1]
```diff
@@ -351,12 +466,24 @@ def request_get_openqa_jobs(self, req):
...
+                    u = TARGET_REPO_SETTINGS[prj]
+                    s = u['settings']
+                    ret += self.openqa.openqa_request(
+                        'GET', 'jobs',
+                        {
+                            'distri': s['DISTRI'],
+                            'version': s['VERSION'],
+                            'arch': s['ARCH'],
+                            'flavor': s['FLAVOR'],
+                            'build':  self.detect_build_for_target(u),
```
Comment (lnussel) [inline openqa-maintenance.py]: this function has a side effect of actually creating jobs. quite dirty in a function that is meant to only check

Comment (coolo) [inline openqa-maintenance.py]: well, the function is just badly named :)

**Comment by coolo** [comment]
https://build.suse.de/request/show/118318 shows a comment for full review

**Comment by lnussel** [comment]
if it works fine with me

**Pull request merged**